libnmstate/nm/wired.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from libnmstate import nmclient
+
+
+def get_info(active_connection):
+    """
+    Provides the current active values for an active connection.
+    """
+    info = {}
+    # FIXME: Maybe use
+    # active_connection.get_devices()[0].get_applied_connection(0) to get the
+    # real runtime connection...
+    try:
+        remote_connection = active_connection.get_connection()
```

Comment:
Seems we just use it as a "connection" and not as a "remote" one, but it is a nit.

Reply:
In NM it is a RemoteConnection (as opposed to a SimpleConnection that `dev.get_applied_connection(0)` returns.